Commands/Config/PresetCommand.cs

```diff
@@ -0,0 +1,117 @@
+using ArtcordBot.Helpers;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.SlashCommands.ArgumentModifiers;
+using DSharpPlus.Entities;
+
+
+namespace ArtcordBot.Features.ConfigCommands
+{
+    public partial class ConfigCommandsGroup
+    {
+        [Command("addPreset")]
+        public async Task AddPreset(CommandContext ctx, string name, string? channels = null, string? members = null)
+        {
+            var embed = MessageHelpers.GenericAddedPresetEmbed(
+                $"Preset {name} Added!",
+                $"Channels: {channels}\nMembers: {members}"
+            );
+
+            channels ??= "None";
+            members ??= "None";
+
+            var preset = await _guildPresetService.GetPresetAsync(ctx.Guild!.Id, name);
+
+            if (preset is not null)
+            {
+                embed = MessageHelpers.GenericErrorEmbed(
+                    title:"Preset Name Taken!",
+                    message:$"The preset name ``{name}`` is already in use. Please choose a different name."
```

Comment:
```suggestion
                    message:$"The preset name `{name}` is already in use. Please choose a different name."
```
A single backtick is enough for discord inline code blocks